### PR TITLE
Flux/experimental chain

### LINKED ISSUE #998: Document 'experimental.chain' function
PR: https://github.com/influxdata/flux/pull/2781/file

### PR DESCRIPTION
**Pull request by sanderson**
Closes #998 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: experimental.chain() function
+description: >
+  The `experimental.chain()` function ...
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:4]: ... intentional?

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+menu:
+  v2_0_ref:
+    name: experimental.chain
+    parent: Experimental
+weight: 302
+---
+
+The `experimental.chain()` function forces two queries in a single Flux script
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:12]: forces >> joins?

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+to run sequentially and outputs the results of the second query.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:13]: if 2 queries are running sequentially, why only outputs of second?

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+The Flux execution planner typically executes multiple queries in a single script in parallel.
+Running the queries sequentially ensures any dependencies the second query has on
+the results of the first query are met.
+
+##### Applicable use cases:
+- Writing to and bucket and querying the written data in a single Flux script or
```
Comment (pierwill) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:19]: *a bucket*

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+menu:
+  v2_0_ref:
+    name: experimental.chain
+    parent: Experimental
+weight: 302
+---
+
+The `experimental.chain()` function forces two queries in a single Flux script
+to run sequentially and outputs the results of the second query.
+The Flux execution planner typically executes multiple queries in a single script in parallel.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:14]: what's the Flux execution planner - thing that executes a query?

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+Running the queries sequentially ensures any dependencies the second query has on
+the results of the first query are met.
+
+##### Applicable use cases:
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:18]: drop colon f header

File: content/v2.0/reference/flux/stdlib/experimental/chain.md (hunk 4 above)
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:19]: Writing to a bucket...?

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+- Writing to and bucket and querying the written data in a single Flux script or
+  [InfluxDB task](/v2.0/process-data/get-started/).
+- Forcing the order of query execution in testing scenarios.
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:21]: Why forcing? would "Order of query..." work?

File: content/v2.0/reference/flux/stdlib/experimental/chain.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+### first
+The first query to execute.
+
+_**Data type:** Stream of tables_
+
+### second
+The second query to execute.
+
+_**Data type:** Stream of tables_
```
Comment (kelseiv) [inline content/v2.0/reference/flux/stdlib/experimental/chain.md:44]: stream of tables a data type?

**Pull request merged**